financialaid/views_test.py

```diff
@@ -128,24 +131,43 @@ def test_income_validation_user_not_enrolled(self):
         self.program_enrollment.save()
         self.assert_http_status(self.client.post, self.request_url, status.HTTP_400_BAD_REQUEST, data=self.data)
 
-    def test_income_validation_currency_not_usd(self):
+    def test_income_validation_currency_not_usd_gto(self):
         """
-        Tests IncomeValidationView post with a currency that is not USD
+        Tests FinancialAidRequestView post with a currency that is not USD with exchange rate greater than 1
```

Comment:
The issue was that we should be using `income_usd` to calculate the tier, but instead we used `original_income`, and since the original test for currency conversion didn't convert to a value that would change which tier they would have been assigned with `original_income`, we never caught this bug.

I'll leave it in for now!